clients/js/test/add.collections.test.ts

```diff
@@ -120,3 +121,25 @@ test("should error on empty embedding", async () => {
     expect(e.message).toMatch("got empty embedding at pos");
   }
 });
+
+
+if (!process.env.OLLAMA_SERVER_URL) {
+  test.skip("it should use ollama EF, OLLAMA_SERVER_URL not defined", async () => {
+  });
+} else {
+  test("it should use ollama EF", async () => {
+    await chroma.reset();
+    const embedder = new OllamaEmbeddingFunction({ url: process.env.OLLAMA_SERVER_URL || "http://127.0.0.1:11434/api/embeddings", model: "llama2" })
```

Comment:
You should set the model to nomic-embed-text. It has far better results in a tiny fraction of the time. Don't use llama2 for embeddings.

Reply:
@technovangelist, Thanks for the suggestion. I will do so.